### ISSUE TITLE
ButtonGroup buttons inconsistent/incorrect styling

### ISSUE THREAD

**Issue by kennylam** [carbon-design-system/ibm-dotcom-library#812]
## Detailed description

> Describe in detail the issue you're having.

`ButtonGroup` secondary button does not have correct styling for `:focus`

<img width="326" alt="Screenshot 2019-12-11 at 8 44 56 AM" src="https://user-images.githubusercontent.com/909118/70626697-e9657f00-1bf2-11ea-9c31-6927b5a6e7ab.png">

and `:active + :hover` states

<img width="331" alt="Screenshot 2019-12-11 at 8 46 14 AM" src="https://user-images.githubusercontent.com/909118/70626743-f97d5e80-1bf2-11ea-9301-724cf1d4be85.png">

## Notes
https://ibmdotcom-react-experimental.netlify.com/?path=/story/buttongroup--default

**Comment by annawen1** [comment]
@photodow mentioned this in my leadspace centered PR about this issue:

> I've already raised the accessibility concern with Carbon for that button, but they haven't responded yet.
carbon-design-system/carbon#4404

**Comment by RobertaJHahn** [comment]
Decision made in the Dec 11 Traige review meeting to override until Carbon fixes this issue.  Set as a Sev 1.

**Comment by RobertaJHahn** [comment]
@oliviaflory Please provide design specs.

**Comment by RobertaJHahn** [comment]
@mattrosno We have classified this issue as a sev 1.  We would prefer not to do the override if Carbon can fix this issue soon. Do you have an ETA?

**Comment by RobertaJHahn** [comment]
Updating to the latest Carbon dependency might address the issue, but needs additional investigation.

**Comment by oliviaflory** [comment]
Button specs for developer reference
![tertiary-button](https://user-images.githubusercontent.com/45692622/70643966-fd6ba980-1c0f-11ea-906c-ac8fa308e57b.png)

**Comment by annawen1** [comment]
Looks like this issue has been fixed.

`:focus` state
<img width="738" alt="Screen Shot 2019-12-19 at 12 54 02 PM" src="https://user-images.githubusercontent.com/54281166/71197161-02a7a480-225f-11ea-9f8b-6cd5521b29c0.png">

`:active` + `:hover` state
<img width="727" alt="Screen Shot 2019-12-19 at 12 54 26 PM" src="https://user-images.githubusercontent.com/54281166/71197178-0affdf80-225f-11ea-8a55-340caac0a21a.png">

ButtonGroup doesn't have a theme applied to it currently, so it is displaying the white theme styling. I applied the carbon dark theme locally to confirm with this ticket that buttonGroup was fixed.
